joomla/joomla-cms
# [5.2] Change Alert Colors to Accessible Colors (AA) for Quickicons

Changed Colors of Quickicons
Added chosen colors to scss vars

Pull Request for PR https://github.com/joomla/joomla-cms/pull/38813

### Summary of Changes
I changed the colors of the Quickicons to accessible colors (AAA) and followed this article for recommendations:
https://medium.com/handsome-perspectives/a-guide-to-color-accessibility-in-product-design-aa3e8919be0


### Testing Instructions
I am not sure if it's ready to test yet.
If we have accessible colors for alerts they should be consistent through the system but it seems that the colors are managed also in an outside repo (https://github.com/joomla-projects/custom-elements), I am not sure how to change them there and test it.
Also I do not understand the logic behind how it was done before. I wanted to use the scss variables but they can't be used with css variables so I put in the values static. Which I think is wrong because the colors should be managed by scss variables in a central file. 

### Actual result BEFORE applying this Pull Request
Note that I changed the class of the Privacy Request Icon to simulate the warning color.

<img width="554" alt="grafik" src="https://user-images.githubusercontent.com/828371/193407008-ddbd829f-4450-48e5-ae61-a4f2dc85be5d.png">


### Expected result AFTER applying this Pull Request

Note that I changed the class of the Privacy Request Icon to simulate the warning color.

![grafik](https://user-images.githubusercontent.com/828371/195994272-8bc0dd30-fa4b-468c-8d1e-f0dc5d3310cc.png)

Note that I changed the class of the left bottom quick-icon to warning for testing purposes.
### Documentation Changes Required
